wolfssl-sys/src/lib.rs

```diff
@@ -40,5 +41,27 @@ mod tests {
             // Check that Kyber was enabled
             assert_eq!(res, WOLFSSL_SUCCESS as c_int);
         }
+
+        // Test ML-KEM is available
```

Comment:
You can use test-case instead of repeating the test case.

Example:
https://github.com/expressvpn/wolfssl-rs/blob/7a1e3d1933541d7a34009d24a9e4c605296ac6e3/wolfssl/tests/ping_pong.rs#L247